tools/src/main/scala/scala/scalanative/linker/Reach.scala

```diff
@@ -219,6 +224,8 @@ class Reach(config: build.Config, entries: Seq[Global], loader: ClassLoader) {
                 update(Rt.JavaHashCodeSig)
               case sig if sig.isMethod || sig.isCtor || sig.isGenerated =>
                 update(sig)
+              case sig if sig.isClinit =>
+                update(sig)
```

Comment:
It looks like this could be merged with the previous `case`, just adding `|| sig.isClinit`.